libs/logic-apps-shared/src/designer-client-services/lib/base/search.ts

```diff
@@ -209,8 +217,7 @@ export abstract class BaseSearchService implements ISearchService {
       };
       // const response = await this.batchAzureResourceRequests(uri, queryParameters);
       const response = await this.getAzureResourceRecursive(uri, queryParameters);
-
-      return response;
+      return this.removeUnsupportedOperations(response);
```

Comment:
Don't do it in custom apis.. those are user deployed and managed and no need to run a iteration on it when we should not filter here. 
We should only do for managed apis